public/app/features/alerting/unified/components/rule-editor/rule-types/RuleType.tsx

```diff
@@ -20,6 +20,10 @@ export interface SharedProps {
   onClick: (value: RuleFormType) => void;
 }
 
+const isClickEvent = (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
+  return e.detail !== 0;
```

Comment:
Maybe add a comment that this means pressing Enter? or link a reference to the issue it solves so we know in the future

Reply:
good idea 👍

Reply:
Good idea adding a comment 👌